packages/calcite-components/src/components/fab/fab.e2e.ts

```diff
@@ -148,4 +149,40 @@ describe("calcite-fab", () => {
       });
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      themed(html`<calcite-fab></calcite-fab>`, {
+        "--calcite-fab-background-color": {
+          targetProp: "backgroundColor",
```

Comment:
For these targetProperties, I think they should target `--calcite-button-background-color` instead of backgroundColor and using shadowSelector. Because they are currently testing internals of the button component which is out of scope for these tests.